src/classes/handlers/EconomyHandler.ts

```diff
@@ -0,0 +1,158 @@
+import FluorineClient from '@classes/Client';
+import r from 'rethinkdb';
+import { EconomyUser } from 'types/economyUser';
+import { SettingsType } from 'types/settings';
+export default class EconomyHandler {
+    client: FluorineClient;
+    constructor(client: FluorineClient) {
+        this.client = client;
+    }
+    async get(user: string, guild: string) {
+        const data = (await r
+            .table('economy')
+            .get(`${user}-${guild}`)
+            .run(this.client.conn)) as EconomyUser;
+        return data?.balance || { wallet: 0, bank: 0 };
+    }
+    async add(user: string, guild: string, amount: number) {
+        const userObj: any = await r
+            .table('economy')
+            .get(`${user}-${guild}`)
+            .run(this.client.conn);
+        if (!userObj) {
+            return r
+                .table('economy')
+                .insert({
+                    id: `${user}-${guild}`,
+                    balance: { wallet: amount, bank: 0 }
+                })
+                .run(this.client.conn);
```

Comment:
this code is used a lot in the handler, consider making it its own method.

Reply:
uhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh